packages/app-mobile/utils/fs-driver-rn.ts

```diff
@@ -260,6 +267,52 @@ export default class FsDriverRN extends FsDriverBase {
 		throw new Error(`Not implemented: md5File(): ${path}`);
 	}
 
+	public async tarExtract(_options: any) {
+		throw new Error('Not implemented: tarExtract');
+	}
+
+	public async tarCreate(options: any, filePaths: string[]) {
+		const cwd = options.cwd; // TODO: What if this is undefined?
```

Comment:
To-do: Handle the case where cwd is undefined (or at least throw an exception).

Reply:
Yes throwing an exception would be fine, if you expect cwd to always be defined. Although if it's q required parameter, why is it called "options"?

Reply:
I've changed this to default to [`RNFS.DocumentDirectoryPath`](https://github.com/itinance/react-native-fs#file-creation) if `cwd` is not provided.